app/User.php

```diff
@@ -242,7 +242,10 @@ public function scopeVerified($query)
     */
     public function groups()
     {
-        return $this->belongsToMany(\App\Group::class, 'membership')->where('membership.membership', '>=', Membership::MEMBER)->orderBy('name')->withTimestamps();
+        return $this->belongsToMany(\App\Group::class, 'membership')
```

Comment:
Just wondering if the removal of the where clause to limit to active members will break something somewhere else. I'll check it out, because it's a good idea to remove it anyway.

Reply:
will revert it in a follow up commit